model/client.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+const uuid = require('node-uuid');
+
+
+const Client = module.exports = function(socket) {
+  this.socket = socket;
+  this.nickname = `user:${Math.floor(Math.random()*10)}${Math.floor(Math.random()*10)}`;
```

Comment:
Quick hint `Math.floor(Math.random()*99)` should give you a number between 0 and 99.